src/Microsoft.DotNet.Wpf/tests/UnitTests/WindowsBase.Tests/Resources.Designer.cs

```diff
@@ -0,0 +1,71 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
```

Comment:
Guess this wasn't supposed to be checked in as per #10023?

Reply:
Yeah, got caught in trying to maintain a chain of PRs, good catch.